survey_form/index.html

```diff
@@ -45,6 +45,35 @@
 
     <form class="form">
       <p class="form__tittle">Lets us now how we can improve freeCodeCamp</p>
+
+      <div class="form__item">
+        <label for="name">Name: </label>
```

Comment:
:+1: za `for` - z punktu widzenia a11y, kliknięcie labela powinno focusować input :)